lib/ui/window/platform_configuration.cc

```diff
@@ -308,6 +308,15 @@ void PlatformConfiguration::ReportTimings(std::vector<int64_t> timings) {
                                                }));
 }
 
+Window* PlatformConfiguration::get_window(int window_id) {
```

Comment:
Yeah I believe so. 

A bit of history: At first the entire `ui` folder only has a `Window` class, then `PlatformConfiguration` is extracted out to be the singleton class, with what's left of `Window` being owned by `PlatformConfiguration`.

I don't want to fix the naming yet because it has a larger impact (the folder name), and also that we're still in the transition period.